### ISSUE TITLE
xoauth token command outputs JSON to stderr instead of stdout

### ISSUE THREAD

**Issue by krsadler** [XeroAPI/xoauth#12]
When using the token command the output goes to stderr instead of stdout which is unexpected. For example:

`xoauth token my-alias-name`

You can still capture the JSON output by redirecting stderr to a file instead:

`xoauth token my-alias-name 2> my-token.json`

**Comment by NathanBeddoeWebDev** [comment]
Yes, I also noticed this when using Node.JS child_process.exec_file, that it wasn't going to the expected parameter.
